sidecar/proxy/envoy/manager.go

```diff
@@ -0,0 +1,623 @@
+// Copyright 2016 IBM Corporation
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package envoy
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/amalgam8/amalgam8/pkg/api"
+	"github.com/amalgam8/amalgam8/sidecar/identity"
+)
+
+// EnvoyConfigPath path to envoy config file
+const EnvoyConfigPath = "/etc/envoy/envoy.json"
+
+const EnvoyLogFormat = "" +
+	"{" +
+	"\"status\":\"%RESPONSE_CODE%\", " +
+	"\"timestamp_in_ms\":\"%START_TIME%\", " +
+	"\"request_time\":\"%DURATION%\", " +
+	"\"upstream_response_time\":\"%RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)%\", " +
+	"\"$a8_trace_key\":\"NEED\", " +
```

Comment:
remove this line for the moment. Envoy has X-Request-ID which we can piggyback on for the moment.